hlc_core/CfgMagazines.hpp

```diff
@@ -1,79 +1,87 @@
-class CfgMagazines {
+class CfgMagazines 
+{
     class 30Rnd_556x45_Stanag;
 
-    class hlc_30rnd_556x45_EPR : 30Rnd_556x45_Stanag{
+    class hlc_30rnd_556x45_EPR : 30Rnd_556x45_Stanag
+    {
         ammo = "HLC_556NATO_EPR";
         count = 30;
         descriptionshort = "Caliber: 5.56x45mm NATO M855A1 EPR<br />Type: Reverse Drawn Full Metal Jacket Exposed Penetrator (Enhanced Performance Round)<br />Rounds: 30";
         displayname = "STANAG Mag (EPR) 5.56mm";
         initspeed = 974.8;
-        mass = 8;
+        mass = 11.5;                                // no reference found

```

Comment:
Why are these heavier than BI Default STANAG mags?

Reply:
BI, simply said, uses volume to find a good load balance.
I and ACE mod, use representive weight/mass, that results from that BI mass value
The mass of 11.5 masspoints equals 522 gramm.

Reply:
I get that much, use ACE myself, but something to keep in mind is that not everyone does, and having them at a different mass point without ACE means that there's an artificially created handicap to running what are effectively the same mags running the same caliber.

Reply:
If the niarms magazine would be too "heavy" for whatever container they use, They can still use the BI magazine. Anyways, the mass change with this is comparatively small. ( in comparison to an  m60 belts e.g. ), In a loadout with 10 magazines it would make a difference of 35 points.
I would like to make them compatible with ACE, so that - in the future - a universal mass standard !can! be established for modders, to build their mods upon the principle of weight, instead of volume. I see this as the only way to find a sustainable cross-mod standard.
Most weapons can be regarded as having a BI companion/compatibility, for me there is no point in changing just those to correct weights, that no one will realize the increase of mass with.